1. Desktop app calls Web API/TodoListService/Startup.cs

```diff
@@ -46,23 +46,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddAuthentication(AzureADDefaults.BearerAuthenticationScheme)
-                .AddAzureADBearer(options => Configuration.Bind("AzureAd", options));
-            services.Configure<JwtBearerOptions>(AzureADDefaults.JwtBearerAuthenticationScheme, options =>
-            {
-			    // This is an Azure AD v2.0 Web API
-                options.Authority += "/v2.0";
-				
-				// The valid audiences are both the Client ID (options.Audience) and api://{ClientID}
-                options.TokenValidationParameters.ValidAudiences = new string[] { options.Audience, $"api://{options.Audience}" };
-
-                // Instead of using the default validation (validating against a single tenant, as we do in line of business apps),
-                // we inject our own multitenant validation logic (which even accepts both V1 and V2 tokens)
-                options.TokenValidationParameters.IssuerValidator = AadIssuerValidator.ForAadInstance(options.Authority).ValidateAadIssuer;
+            services.AddProtectWebApiWithMicrosoftIdentityPlatformV2(Configuration)
+                    ;
```

Comment:
; on newline

Reply:
Thanks @bgavrilMS. Will address in next pr